recipes/khronos-opencl-icd-loader/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "cb4acb99df3441e147bd859c20d9c3533ec95ce8" %}
+{% set sha256 = "e1cc1697d33d8662a0f082603ac8192b5feaa3f97afd3b4ff929ed41fd2fea13" %}
+
+package:
+  name: khronos-opencl-icd-loader
+  version: 2016.10.12
+
+source:
+  fn: {{ version }}.zip
+  url: https://github.com/KhronosGroup/OpenCL-ICD-Loader/archive/{{ version }}.zip
+  sha256: {{ sha256 }}
+
+  patches:
+  # Khronos don't ship those headers, but they require them to build.
+  # See https://github.com/KhronosGroup/OpenCL-ICD-Loader/blob/master/inc/README.txt
+  - add-khronos-headers.patch
+  # Khronos doesn't have an install target in their CMake. Add one.
+  - add-cmake-install.patch
+
+build:
+  number: 0
+
+  # We've got ocl-icd on Linux and the built-in loader on OS X.  No need to use
+  # this ... fantastic piece of software ... anywhere else.
+  skip: true  # [not win]
+
+requirements:
+  build:
+    - toolchain
```

Comment:
This is unnecessary.  At the end of the day the toolchain only changes, and in an irreproducible way since it is not versioned, the builds on OS X.